tests/forms/DropdownFieldTest.php

```diff
@@ -224,6 +224,31 @@ public function testNumberOfDisabledOptions() {
 	}
 
 	/**
+	 * The Field() method should be able to handle arrays as values in an edge case. If it couldn't handle it then
+	 * this test would trigger an array to string conversion PHP notice
+	 *
+	 * @dataProvider arrayValueProvider
+	 */
+	public function testDropdownWithArrayValues($value) {
+		$field = $this->createDropdownField();
+		$field->setValue($value);
+		$this->assertInstanceOf('HTMLText', $field->Field());
```

Comment:
I think we can have a more thorough assertion here to test the behaviour is as expected (ie: value is correctly set) rather than simply that there is no error.